packages/contracts/test/src/token/CBLTest.t.sol

```diff
@@ -3,8 +3,8 @@
 pragma solidity ^0.8.20;
 
 import { Test } from "forge-std/Test.sol";
-import { CBL } from "../../src/token/CBL.sol";
-import { DeployCBLToken, CBLTokenParams } from "../../script/DeployCBLToken.s.sol";
+import { CBL } from "../../../src/token/CBL.sol";
```

Comment:
FYI there are utility remappings built in to reduce the need for those import paths! So the `../../../src` can be replaced with `@credbull`

Not required though.